### DISCUSSION TITLE
How to deploy on directadmin?

### DISCUSSION THREAD

**Discussion by duytv14** [parcelvoy/platform#496] [Q&A]
I have run the command "lerna run build" and then copy the build folder of "platform" and "ui" to directadmin.
But when I run it I get the following error.
![image](https://github.com/user-attachments/assets/0deec5e7-d750-42a1-8f40-b42e8db90bcb)
logs
App 1768053 output: /etc/profile: line 86: /etc/init.d/msm_profile: No such file or directory
App 1768053 output: node:internal/modules/cjs/loader:1146
App 1768053 output:   throw err;
App 1768053 output:   ^
App 1768053 output: 
App 1768053 output: Error: Cannot find module 'knex'
App 1768053 output: Require stack:

**Comment by pushchris** [comment]
Not sure I can provide direct guidance on how to use this tool as I've never heard of it before, but the error message insinuates that you didn't copy over the files or node modules correctly. Unless running in development we do not encourage running this directly from checking out the code, you should instead use the Docker images. If that is not possible for you, your best bet would be to look at exactly what the Dockerfile is doing and mimic the steps for your setup. Each one is important.
